tests/tests.h

```diff
@@ -0,0 +1,159 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2004 - 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_tests_h
+#define dealii_tests_h
+
+#include <deal.II/base/logstream.h>
+#include <deal.II/base/mpi.h>
+
+#include <fstream>
+
+DEAL_II_NAMESPACE_OPEN
+
+// given the name of a file, copy it to deallog
+// and then delete it
+void
+cat_file(const char *filename)
```

Comment:
If you think this function is useful you can mention it in https://github.com/dealii/dealii/pull/13033.

Reply:
This function is used in `struct MPILogInitAll` so must be declared otherwise one gets a build error.

Reply:
I write a short comment about that in dealii/dealii#13033.